src/components/modal/v2/parts/ContentWrapper.jsx

```diff
@@ -21,15 +21,21 @@ const ContentWrapper = () => {
 
     // Add views to productView object where the keys are the product name and the values are the view component
     const productView = {
-        PAY_LATER_LONG_TERM: <LongTerm {...useContent(product)} />
+        PAY_LATER_LONG_TERM: <LongTerm {...useContent(product)} />,
+        NI: <NI {...useContent(product)} />
```

Comment:
Once the sticky scroll change gets merged in, let's pass in the contentBodyRef.
```suggestion
        NI: <NI {...useContent(product)} contentBodyRef={contentBodyRef} />
```